### PR TITLE
Update package lock with 2.0.0 version

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Update package lock with 2.0.0 version and release workflow
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| Sponsor company   | ~
| How to test?      | ~

### CODE REVIEW

**Review by mattgoud — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
